components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -438,6 +438,7 @@ SUBROUTINE p3_main(qc,nc,qr,nr,th,qv,dt,qitot,qirim,nitot,birim,   &
     ! so that they can be written as ouput.  NOTE TO C++ PORT: This variable is entirely optional and doesn't need to be
     ! included in the port to C++, or can be changed if desired.
     real(rtype), intent(out),   dimension(its:ite,kts:kte,49)   :: p3_tend_out ! micro physics tendencies
+    real(rtype), intent(in),    dimension(its:ite,2)            :: col_location
```

Comment:
Should dimension be (its:ite,3)?

Reply:
oh yes, they should be, I'm surprised it still works...  I'll fix this